WireAuthentication/Sources/WireAuthenticationUI/Views/Login/ReloginViaEmail/ReloginViaEmailView.swift

```diff
@@ -0,0 +1,234 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireAuthenticationAPI
+import WireDesign
+import WireNetwork
+import WireReusableUIComponents
+
+package protocol ReloginViaEmailFactory {
+
+    @MainActor var viewModel: ReloginViaEmailViewModel { get }
+
+    @MainActor
+    func verifyLoginView(
+        email: String,
+        password: String,
+        proxyCredentials: ProxyCredentials?
+    ) -> VerificationCodeView
+
+    @MainActor
+    func noHistoryView(result: AuthenticationResult) -> NoHistoryView
+
+}
+
+package struct ReloginViaEmailView: View {
```

Comment:
This was mostly taken from `LoginViaEmailView` with bits removed. I think we can find some ways to reduce the duplication but I'd say it's out of scope of this PR.